src/hooks/user/useGetNationalities.ts

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from 'react';
```

Comment:
This seems fine but I am concerned about the naming of the hooks and the files in general. For example I see that everywhere else we use this pattern `useUserData` or just `useFeatures` but now you are including this `Get` keyword in the naming of the hooks and files. Not sure if this is necessary because we use the hooks to query something and it is quite understandable that it is for getting some data. You can probably use just `useNationalities` and `useCountries`. Let me know what do you think about this. Just mentioning because I want to keep the consistency.

Reply:
These of course should be called like that